test.sh

```diff
@@ -70,6 +92,24 @@ case $mode in
 	  multiqc_config=${PWD}/vdb_shotgun/multiqc_config.yaml \
 	  nshards=2
       ;;
+  denoise_pooled_se )
+      snakemake \
+	  $commonargs \
+	  --singularity-args "-B ${PWD},/data/brinkvd/,/lila/$PWD/,/lila/data/brinkvd/,/scratch/" \
+	  --directory tmpdenoisese/ \
+	  --config \
+	  stage=denoise \
+	  sample=473  \
+	  pool="pool1" \
+	  manifest=$PWD/tmppre/preprocess/pool1_manifest.tsv \
+	  oligos=$PWD/.test/amplicon/test_input/pool1059.oligos \
+	  R1=$R1 \
+	  R2=$R2 \
```

Comment:
Is this R2 supported to be here?

Reply:
(as in I wasn't sure if you were testing how it performed if you passed in an R2 for single end!)

Reply:
removed and added a check, thanks!